config/swagger.yaml

```diff
@@ -402,6 +402,434 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security:
+  /accounts/{pubkey}:
+    get:
+      tags:
+        - external
+        - gossip
+      operationId: GetAccountByPubkey
+      description: 'Get an account by public key'
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: pubkey
+          description: 'The public key of the account'
+          required: true
+          type: string
```

Comment:
Is this base58check encoded (with account prefix)? Maybe this can be reflected in type?

Reply:
Same as above. We should take care of it in following PRs